packages/material-ui/src/locale/index.js

```diff
@@ -712,6 +769,9 @@ export const hyAM = {
 };
 
 export const idID = {
+  localization: {
+    formatNumber: formatNumber('id-ID'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@fuadinaqi Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).